src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,111 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
-
+import { getConnect } from '../../api/goods';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onFindFilm }) => {
+  const [title, setTitle] = useState('');
```

Comment:
It's better to set Error here instead of the title because I see "Can't find a movie" from the start even when I have put nothing